xml/hwbp_registers.xml

```diff
@@ -145,6 +145,34 @@
         <field name="0" bits="2:0" access="R" reset="0" />
     </register>
 
+    <register name="Machine Context" short="mcontext" address="0x7a8">
```

Comment:
is this address appropriate for a register accessible in M and Debug mode?

Reply:
Yes. 0x7a0 -- 0x7af are reserved for M-mode "standard read/write debug CSRs".